runtime/src/snapshot_utils.rs

```diff
@@ -2953,10 +2947,10 @@ pub fn purge_old_bank_snapshots(
             .rev()
             .skip(num_bank_snapshots_to_retain)
             .for_each(|bank_snapshot| {
-                let r = remove_bank_snapshot(bank_snapshot.slot, &bank_snapshots_dir);
+                let r = purge_bank_snapshot(&bank_snapshot.snapshot_dir);
                 if r.is_err() {
```

Comment:
Line became a bit shorter, maybe we can be a bit more concise as it's one line now.

(tested - clippy and fmt are happy)
```suggestion
                if purge_bank_snapshot(&bank_snapshot.snapshot_dir).is_err() {
```